packages/esm-patient-search-app/src/config-schema.ts

```diff
@@ -12,6 +12,12 @@ export const configSchema = {
       _default: false,
       _description: 'Whether to show recently searched patients',
     },
+    disableTabletSearchOnKeyUp: {
+      _type: Type.Boolean,
+      _default: false,
+      _description:
+        'Disable the default "keyup search" for instant patient search as typing concludes on tablet devices',
```

Comment:
If this Config is _not_ tablet specific, then the description should not be tablet-specific either ;)